src/services/nativeAuth/methods/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './getTokenExpiration';
-export * from './getToken';
+export * from './getTokenExpirationTime';
```

Comment:
I think this function should not sit here but near LogoutListener

Reply:
Moved in a separate helpers folder inside the new hook using it. 👍